README.rst

```diff
@@ -450,6 +450,69 @@ The following options (``[flags]``) are supported:
   Defaults to ``./instances.yml`` (or to the value of the "cfg"
   parameter in the Cartridge `configuration file <Overriding default options_>`_).
 
+  The ``instances.yml`` file contains parameters for starting Cartridge
+  application instances and is placed in the application root directory.
+  These parameters are then passed to the `cartridge.cfg <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_api/modules/cartridge/#cfg-opts-box-opts>`_
+  function.
+
+  Example of the ``instances.yml`` file:
+
+  .. code-block:: yaml
+
+      myapp.router:
+        advertise_uri: localhost:3301
+        http_port: 8081
+
+      myapp.s1-master:
+        advertise_uri: localhost:3302
+        http_port: 8082
+
+  Parameters that can be specified in ``instances.yml`` are of two kinds:
+
+  * `cartridge.cfg <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_api/modules/cartridge.argparse/#tables>`_
+    parameters
+  * custom parameters.
+
+  Among the ``cartridge.cfg`` parameters, ``advertise_uri`` is required while
+  others are optional.
+
+  .. note::
+
+     Some of the ``cartridge.cfg`` parameters are exceptions: they passed by
+     Catridge CLI itself and cannot be specified in the ``instances.yml`` file:
+
+     * ``workdir``
+     * ``console_sock``
+     * ``pid_file``.
+
+  Regarding the custom parameters, they should be defined in the application
+  code first. For example:
+
+  .. code-block:: lua
+
+      cartridge.cfg({my_param = 'default-value'})
+
+      local argparse = require('cartridge.argparse')
+      local my_param = argparse.get_opts({my_param='string'})
+
+  And then you can mention your custom parameters in ``instances.yml``:
+
+  .. code-block:: yaml
+
+      myapp.router:
+        advertise_uri: localhost:3301
+        http_port: 8081
+        my_param: 'Hello, world'
+
+  At the application start (``myapp`` in our example above), Cartridge CLI
+  passes the path to configuration file (``TARANTOOL_CFG=instances.yml``)
```

Comment:
It can be rewritten by `--cfg` option of `cartridge start`.

Reply:
I think @veod32 was describing the mechanics, but it's still unclear.

Reply:
Removed. I think this is not for description of an option and should be explained in more generic topics related to app start.